vms/proposervm/vm_test.go

```diff
@@ -478,15 +478,21 @@ func TestCoreBlockFailureCauseProposerBlockParseFailure(t *testing.T) {
 		return nil, errMarshallingFailed
 	}
 
+	emptyParentBlockSig := []byte{}
```

Comment:
nit: should this be a package level variable, to avoid reinstantiating it in different tests?

Reply:
done.